spec/features/editing_file_attachments/replace_file_attachment_file_spec.rb

```diff
@@ -9,6 +9,16 @@
     and_i_see_the_timeline_entry
   end
 
+  scenario "featured" do
+    given_there_is_an_edition_with_featured_attachments
+    when_i_go_to_change_an_attachment
```

Comment:
Minor: could we combine these steps and visit the page directly? Every page we visit costs up to 1s in our test runs, so it's good to avoid unnecessary page renders if we can. (We should probably apply this in a big sweep!)